### PR TITLE
docs: add Chinese of developer guide

### PR DESCRIPTION
**Pull request by ruquanzhao**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by ruquanzhao** [comment]
I have read the CLA Document and I hereby sign the CLA

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
 ---
-title: Non-code Contribution Guide
+title: 非代码贡献指南
 ---
-You can contribute in any of the following ways that interest you.
+你可以用以下任何感兴趣的方式贡献。
 
-## Contributing Use Cases and Demos
+## 贡献用户实例和 Demo
```
Comment (elliotxx) [inline i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md:6]: s/实例/示例

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
+## 提议增强特性
 
-If you have ideas to improve Karpor, please submit a [feature request](https://github.com/KusionStack/karpor/issues/new?assignees=&labels=kind%2Ffeature&projects=&template=enhancement.yaml).
+如果你有提升 Karpor 的好点子，请提交 [特性请求](https://github.com/KusionStack/karpor/issues/new?assignees=&labels=kind%2Ffeature&projects=&template=enhancement.yaml)。
 
-## Answering Questions
+## 回答问题
 
-If you have a question and cannot find the answer in the [documentation](https://www.kusionstack.io/karpor/), the next step is to ask on [GitHub Discussions](https://github.com/KusionStack/karpor/discussions).
+如果你有疑问并且在 [文档](https://www.kusionstack.io/karpor/) 中找不到答案，下一步是在 [GitHub 讨论](https://github.com/KusionStack/karpor/discussions) 中提问。
```
Comment (elliotxx) [inline i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md:30]: s/Github 讨论/Github 论坛

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/2-code-contribute.md [hunk 3]
```diff
@@ -1,174 +1,174 @@
...
-### Passing All CI Checks
+### 通过所有 CI 检查
 
-Before merging, all testing CIs should pass:
+在合并之前，所有的测试 CI 都应该通过：
 
-- Coverage should not drop. Currently, the pull request coverage should be at least 70%.
-- Karpor uses a **CLA** for the contributor agreement. It requires you to sign for every commit before merging the pull request.
+- 覆盖率不应该下降。当前，pull request 的覆盖率应当至少为 70%。
+- Karpor 使用 **CLA** 作为贡献者协议。它要求你在合并 pull request 之前为每个 commit 签署。
```
Comment (elliotxx) [inline i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/2-code-contribute.md:164]: s/它要求你在合并 pull request 之前为每个 commit 签署。/它要求你在第一次合并 pull request 之前签署。

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
 
-- MAJOR version when you make incompatible API changes.
-- MINOR version when you add functionality in a backwards-compatible manner.
-- PATCH version when you make backwards-compatible bug fixes.
+- MAJOR version：进行不兼容的 API 更改时。
+- MINOR version：以向后兼容模式添加功能时。
+- PATCH version：以向后兼容模式修复漏洞时。
 
-**Pre-release version numbers and build metadata** can be added to the `MAJOR.MINOR.PATCH` as an extension, like `v1.2.3-alpha.0`, `v1.2.3-beta.1`, `v1.2.3-rc.2`, where `-alpha.0`, `-beta.1`, `-rc.2` are pre-release versions.
+**预发布版本号和构建元数据** 可以作为扩展添加到 `MAJOR.MINOR.PATCH` 中，比如 `v1.2.3-alpha.0`, `v1.2.3-beta.1`, `v1.2.3-rc.2`, 其中 `-alpha.0`, `-beta.1`, `-rc.2` 是预发布版本号。
```
Comment (elliotxx) [inline i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md:49]: s/预发布版本号和构建元数据/先行版本号及版本编译信息

详情参考官方说明：https://semver.org/lang/zh-CN/

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md [hunk 5]
```diff
@@ -1,49 +1,49 @@
...
 
-The version format: `MAJOR.MINOR.PATCH`, for example, `v1.2.3`. The version number **incrementing rules** are as follows:
+版本格式为：`MAJOR.MINOR.PATCH`，例如， `v1.2.3`。版本号  **递增规则** 如下：
 
-- MAJOR version when you make incompatible API changes.
-- MINOR version when you add functionality in a backwards-compatible manner.
-- PATCH version when you make backwards-compatible bug fixes.
+- MAJOR version：进行不兼容的 API 更改时。
+- MINOR version：以向后兼容模式添加功能时。
+- PATCH version：以向后兼容模式修复漏洞时。
```
Comment (elliotxx) [inline i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md:47]: 可以参考官方的翻译：https://semver.org/lang/zh-CN/

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/4-commit-conventions.md [hunk 6]
```diff
@@ -1,33 +1,33 @@
 ---
-title: Commit Conventions
+title: Commit 规约
 ---
-## Commit Message Structure
+## Commit 信息结构
 
-Karpor adheres to [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/).
+Karpor 遵循 [规范化 commit](https://www.conventionalcommits.org/en/v1.0.0/)。
```
Comment (elliotxx) [inline i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/4-commit-conventions.md:6]: s/规范化 commit/约定式提交
s/https://www.conventionalcommits.org/en/v1.0.0/https://www.conventionalcommits.org/zh-hans/v1.0.0/

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md (hunk 1 above)
Comment (ruquanzhao) [reply i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md:6]: Done

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md (hunk 2 above)
Comment (ruquanzhao) [reply i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/1-non-code-contribute.md:30]: Done

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/2-code-contribute.md (hunk 3 above)
Comment (ruquanzhao) [reply i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/1-contribution-guide/2-code-contribute.md:164]: Done

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md (hunk 4 above)
Comment (ruquanzhao) [reply i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md:49]: done

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md (hunk 5 above)
Comment (ruquanzhao) [reply i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/1-release-process.md:47]: done

File: i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/4-commit-conventions.md (hunk 6 above)
Comment (ruquanzhao) [reply i18n/zh/docusaurus-plugin-content-docs-karpor/current/4-developer-guide/2-conventions/4-commit-conventions.md]: done

**Review by elliotxx — APPROVED** [review]
lgtm

**Pull request merged**